packages/invoices/src/OutboundShipment/DetailView/reducer.ts

```diff
@@ -85,9 +86,22 @@ export interface OutboundShipmentStateShape {
 
 type InvoiceLinesByItemId = Record<string, OutboundShipmentRow[]>;
 
-const createItem = (
+export const itemToSummaryItem = (item: Item): OutboundShipmentSummaryItem => {
+  return {
+    id: item.id,
+    itemId: item.id,
+    itemName: item.name,
+    itemCode: item.code,
+    itemUnit: item.unitName,
+    batches: [],
+    unitQuantity: 0,
+    numberOfPacks: 0,
+  };
+};
```

Comment:
this is for adding a new item to the invoice - an empty summary item. I was trying to think of a nice way to do this, maybe making pseudo-`OutboundShipmentRow`s from the stock lines instead of using `batchRow` in state, or something like that 🤷

Reply:
I do like the idea of lifting the `batchRow` out, if I understand that correctly

Reply:
I'm not sure if we are on the same page.

Wasn't able to use outbound shipment rows as we needed the available quantity etc from the stock line